LICENSE

```diff
@@ -2,7 +2,7 @@ SHOTGUN PIPELINE TOOLKIT SOURCE CODE LICENSE
 
 Version: 7/07/2013
 
-Shotgun Software Inc. ("Company") provides the Shotgun Pipeline Toolkit,
+Shotgun Software Inc. ("Company") provides the Flow Production Tracking Toolkit,
```

Comment:
Does this mean we need to bump the version for this LICENCE (line 3)? @julien-lang